src/renderer/html_handlebars/hbs_renderer.rs

```diff
@@ -121,12 +121,10 @@ impl HtmlHandlebars {
     }
 
     fn post_process(&self, rendered: String) -> String {
-        let rendered = build_header_links(rendered, "print.html");
-        let rendered = fix_anchor_links(rendered, "print.html");
-        let rendered = fix_code_blocks(rendered);
-        let rendered = add_playpen_pre(rendered);
-
-        rendered
+        let rendered = build_header_links(&rendered, "print.html");
+        let rendered = fix_anchor_links(&rendered, "print.html");
+        let rendered = fix_code_blocks(&rendered);
+        add_playpen_pre(&rendered)
```

Comment:
Well some of the changes might be arguable but these things tend to be subjective. I'm not really attached to this work As I mostly wanted to take clippy and rustfix for a spin. Just tell me which changes you don't like and I'll drop these :)